Imapc pop3c improvements

slusarz opened the pull request.

sirainen commented on source/admin_manual/mailbox_formats/pop3c.rst:
```diff
@@ -0,0 +1,12 @@
+
+The pop3c storage accesses a remote POP3 server as if it were a regular
+(local) Dovecot mailbox format. Dovecot can treat it as a dummy storage or
+optionally a more capable storage.
```
I see this is just copied from imapc, but I don't see how pop3c could be treated as anything else than a dummy storage. :) So just remove this sentence?

Also could say that the the POP3 mailbox is visible as the INBOX folder on Dovecot side.

sirainen commented on source/configuration_manual/mail_location/pop3c/index.rst:
```diff
@@ -0,0 +1,194 @@
+:ref:`Mail user variables <variables-mail_user>` can be used.
+
+See also :ref:`setting-pop3c_password`.
+See also :ref:`setting-pop3c_user`.
```
were we supposed to start using `.. seealso::`?

sirainen commented on source/configuration_manual/mail_location/pop3c/index.rst:
```diff
@@ -0,0 +1,194 @@
+-------------
+
+- Default: <empty>
+- Values: <empty>, ``no``, ``pop3s``, ``starttls``
```
Where are you getting empty from? The default is "no", and empty value is invalid:

```
Invalid -o parameter pop3c_ssl=: Invalid value:
```

sirainen commented on source/configuration_manual/mail_location/pop3c/index.rst:
```diff
@@ -0,0 +1,194 @@
+- Values: :ref:`string`
+
+The user identity to be used for performing a regular LOGIN to the source POP3
+server.
```
IMAP has LOGIN, but pop3 has USER+PASS. So this should say something else than LOGIN. Same elsewhere in this page.

slusarz commented:
Let's hold off on this until #344 is resolved.  I explicitly did not change any of the pop3c settings generation there, since this MR is pending (and will need to be refactored when/if #344 is approved).

slusarz commented:
I've implemented all the suggestions.  However ... these changes should be done as part of the general settings conversion that is happening in #344.  The imapc changes is just part of the conversion anyway.  The pop3c changes are a bit more substantial, but doesn't make any sense to keep the old settings and then deal with merge issues, so I've moved pop3c conversion to a separate commit in #344.  Closing this MR.
